vllm/block.py

```diff
@@ -1,12 +1,36 @@
 """Token blocks."""
-from typing import List
+import weakref
+from collections import defaultdict
+from typing import Dict, List
 
 from vllm.utils import Device
 
 _BLANK_TOKEN_ID = -1
 
 DEFAULT_LAST_ACCESSED_TIME = -1
 
+TOKEN_BLOCKS = List[int]
```

Comment:
Usually this is for constants rather than type names?
```suggestion
TokenBlocks = List[int]
```